### PR TITLE
changed versioning api call from python to graphql

### PR DESCRIPTION
**Pull request by jordanvelezbomba**
main concern is that the post is right (lines 83 & 84)

### CODE REVIEW

File: website/ui/src/apps/index/components/tab_versions_screen.js [hunk 1]
```diff
@@ -3,120 +3,145 @@
...
-		console.log(err);
-                this.setState({isFetching: false, isError: true});
-	    });
+    if (this.state.isFetching) {
+      return;
     }
+    this.setState({ isFetching: true });
+    // const jq = JSON.stringify({ assembly: "GRCh38" });
+    ApiClient.getByPost(
+      "/dataws/ground_level_versions",
```
Comment (NishiPhalke) [inline website/ui/src/apps/index/components/tab_versions_screen.js:84]: why are we still calling python api here ?

**Comment by jordanvelezbomba** [comment]
check lines 73 / 74

File: website/ui/src/apps/index/components/tab_versions_screen.js [hunk 2]
```diff
@@ -3,120 +3,163 @@
...
+      version
+      biosamples {
+        biosample
+        assays {
+          assay
+          experiments
+        }
+      }
     }
+    bigRequests(requests: [{ url: "gs://gcp.wenglab.org/SCREEN/ground_level_versions" }]) {
```
Comment (NishiPhalke) [inline website/ui/src/apps/index/components/tab_versions_screen.js:73]: why are we making this call ?

**Comment by jordanvelezbomba** [comment]
The above changes were made to fix local environment, they should be reviewed and may not be necessary for deployment.

**Pull request merged**